### PR TITLE
Update what-is-an-operator.md

### PR DESCRIPTION
**Pull request by kaovilai**

### CODE REVIEW

**Review by tuminoid — CHANGES_REQUESTED** [review]

File: _faqs/what-is-an-operator.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 question: What is an operator?
 ---
 
-An Operator is a method of packaging, deploying and managing a Kubernetes application. A Kubernetes application is an application that is both deployed on Kubernetes and managed using the Kubernetes APIs and kubectl tooling. You can think of Operators as the runtime that manages this type of application on Kubernetes. If you want to learn more about Operators you can check the CoreOS Operators website [https://coreos.com/operators/](https://coreos.com/operators/)
+An Operator is a method of packaging, deploying and managing a Kubernetes application. A Kubernetes application is an application that is both deployed on Kubernetes and managed using the Kubernetes APIs and kubectl tooling. You can think of Operators as the runtime that manages this type of application on Kubernetes. If you want to learn more about Operators you can check the Operator framework Operators website [https://operatorframework.io/what/](https://operatorframework.io/what/)
```
[the remark this answers is not in the document]
Comment (kaovilai) [reply _faqs/what-is-an-operator.md:5]:
```suggestion
An Operator is a method of packaging, deploying and managing a Kubernetes application. A Kubernetes application is an application that is both deployed on Kubernetes and managed using the Kubernetes APIs and kubectl tooling. You can think of Operators as the runtime that manages this type of application on Kubernetes. If you want to learn more about Operators you can check the Operator framework website [https://operatorframework.io/what/](https://operatorframework.io/what/)
```

**Comment by tuminoid** [comment]
Thanks. We need to fix the deploy tests, but LGTM now.

**Comment by tuminoid** [comment]
@kaovilai the test deploy is fixed. Could you rebase?

**Comment by kaovilai** [comment]
"This branch has no conflicts with the base branch"

Don't see any indication of rebase required?

**Comment by kaovilai** [comment]
I guess to retrigger deploy.. ok

**Comment by tuminoid** [comment]
Rebase was to the checks working in this PR. Please squash the commits and were good to merge.

**Review by tuminoid — CHANGES_REQUESTED** [review]
Commit squashing required.

**Comment by kaovilai** [comment]
github should've figured out a button for this in web ui by now.

**Review by tuminoid — APPROVED** [review]
LGTM.

/cc @Rozzii @lentzi90

**Review by lentzi90 — APPROVED** [review]
/approve

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Pull request merged**